a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -0,0 +1,201 @@
+import test from 'ava';
+import {
+  agops,
+  bankSend,
+  getUser,
+  openVault,
+  adjustVault,
+  closeVault,
+  getISTBalance,
+  queryVstorage,
+  ATOM_DENOM,
+  USER1ADDR,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
```

Comment:
One thing I'd like to see in a long-lived acceptance test of vaults is that vaults that existed before the most recent upgrade continue to be useable.

Apparently all the vaults that were opened in earlier phases were also closed. 
@turadg, can we retroactively add opening a vault to upgrade 11?  It would be kinda nice if a3p gave us a long-lived vault to interact with.

Reply:
Thank you very much for the suggestion! 

I agree that testing vaults that existed prior to an upgrade is a relevant and important test case to include in the `z:acceptance` test phase,  to ensure proper coverage.

With this in mind, I've opened a dedicated [ticket](https://github.com/Agoric/BytePitchPartnerEng/issues/15) to ensure this scenario is included in our backlog of tests to implement.